x-pack/legacy/plugins/ml/public/timeseriesexplorer/components/timeseriesexplorer_no_results_found/timeseriesexplorer_no_results_found.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/*
+ * React component for rendering EuiEmptyPrompt when no results were found.
+ */
+
+import React from 'react';
+
+import { EuiEmptyPrompt } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+export const TimeseriesexplorerNoResultsFound = ({ dataNotChartable, entities }) => (
```

Comment:
Maybe something like `TimeseriesexplorerNoChartData` might be a better name for this, as it covers the case of no model plot or source data being available, as well as there being no results as e.g. the time range is set outside the range of available data.

Reply:
Renamed in 86d5aa8953.